lib/modules/apostrophe-events-pages/views/index.html

```diff
@@ -1,11 +1,69 @@
 {% extends data.outerLayout %}
+{%- import 'apostrophe-pager:macros.html' as pager with context -%}
+{%- import 'apostrophe-events:macros.html' as eventMacros with context -%}
+
 {% block title %}{{ data.page.title }}{% endblock %}
 
 {% block main %}
-  {% for piece in data.pieces %}
-    <div class="apos-event-preview">
-      <h4 class="apos-event-title"><a href="{{ piece._url }}">{{ piece.title }}</a></h4>
-      <h5 class="apos-event-date">{{ piece.start | date('MMMM Do') }} at {{ piece.start | date('h:mma') }}</h5>
+    <div class="apos-event-wrapper">
+	    <div class="apos-event-sidebar">
+	    	{{ apos.singleton(data.page, 'calendar', 'apostrophe-events-calendar', {}) }}
+	    </div>
+	    <div class="apos-event-main">
+	    	<h1>{{ data.page.title }}</h1>
+	    	<div class="apos-event-filters-wrapper">
+	    		{{ eventMacros.renderFilters(data.query, data.piecesFilters) }}
+	    	</div>
+			  {% for piece in data.pieces %}
+			    <div class="apos-event-preview">
+			    	<h5 class="apos-event-date">{{ piece.start | date('MMMM Do') }} at {{ piece.start | date('h:mma') }}</h5>
+			      <h4 class="apos-event-title"><a href="{{ piece._url }}">{{ piece.title }}</a></h4>
+			    </div>
+			  {% endfor %}
+			  <div class="apos-event-pagination">
+			  	{{ pager.render({ page: data.currentPage, total: data.totalPages }, data.url) }}
+			  </div>
+	    </div>
     </div>
-  {% endfor %}
+    
+{% endblock %}
+
+{% block extraHead %}
+<style>
```

Comment:
Why are we embedding a style block? Can't we push a stylesheet for this?

Reply:
backed out